js/debug/job_recommendation.js

```diff
@@ -0,0 +1,55 @@
+(function ($) {
+    var click = function(jobId, recommender){
+      if(!JOBCENTRE.capabilities.localStorage)
+          return;
+
+      var click = {'jobId': parseInt(jobId), 'recommender': recommender, 'timestamp': new Date().toISOString()};
```

Comment:
I think it would be easier to work with epoch (`new Date().getTime()`) instead of the string representation.  That way, you don't have to parse the string to do comparisons.